mgmtsystem_extended_product/readme/ROADMAP.rst

```diff
@@ -0,0 +1,2 @@
+* add filter on Immediate Actions section in Nonconformity form view
+* manage Template Action on statistic analysis
```

Comment:
Can you clearify the roadmap?
TBH I'm not sure I understand it ^^

Reply:
Sorry, it is a typo from the original module.